Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -714,8 +714,8 @@ WebPage::WebPage(PageIdentifier pageID, WebPageCreationParameters&& parameters)
 #endif
 
 #if HAVE(STATIC_FONT_REGISTRY)
-    if (parameters.fontMachExtensionHandle)
-        WebProcess::singleton().switchFromStaticFontRegistryToUserFontRegistry(WTFMove(*parameters.fontMachExtensionHandle));
+    if (parameters.fontMachExtensionHandles.size() > 0)
```

Comment:
No need for > 0  check here since negative sizes are not allowed. Just checking size() will do what we want and is less verbose.